wiki/Client/Release_stream/Lazer/Feature_comparison/en.md

```diff
@@ -0,0 +1,438 @@
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### The PERFECT judgement hit window scales with OD
+
+This used to be a constant +-16 ms regardless of overall difficulty.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | No? |
+
+#### Converts no longer have different hit windows
+
+Beatmaps converted from the osu! game mode into osu!mania had a set of hit windows that were not affected by OD.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes? |
+| Needs further consideration | No? |
+
+#### Audio rate adjustments affect hit windows in osu!mania
+
+In stable, osu!mania reverts changes to hit windows resulting from applications of DT/HT.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | Yes |
+
+### Scoring, performance points and leaderboards
+
+Scoring in lazer has been simplified with the main benefit of not requiring a replay to recompute. Two interchangeable systems have been added: *standardised* and *classic*. Standardised scoring limits score to a maximum of 1,000,000 points + bonus and score multipliers, while classic scoring is similar to standardised, but scaled quadratically with the amount of hit objects in a beatmap. These can be selected from the settings, with all places were score is displayed in-game changing accordingly.
+
+There are also some differences in how much score each hit object and each judgement is responsible for relative to each other.
+
+Features that depend on score such as [score ranking](/wiki/Ranking#score-ranking) and [levels](/wiki/Gameplay/Score/Total_score#level) need further discussion for how they will be affected.
+
+Medals do not get awarded. This is a feature that will be enabled after anti-cheat measures have been implemented, and will need further consideration regarding the change in difficulty of some medals due to other gameplay changes.
+
+Performance points are not integrated into global or beatmap-specific leaderboards, but are instead integrated in a separate version of the website for now: <https://lazer.ppy.sh/>.
+
+### Storyboarding
+
+#### Storyboard editor not implemented
+
+There are [designs](https://www.figma.com/file/ytnnne2TH8Z956Jxhiypqq/Beatmap-Editor-2) for an upcoming built-in storyboard editor similar to the one in stable, but no work on this has started yet.
+
+  - The trigger system is not implemented
+  - Beatmap backgrounds do not have the correct stretch mode for 4:3 storyboards.
+
+## Some cosmetic features are missing
+
+These are features that are planned to come back at some point, but have not due to low priority.
+
+- Beatmap and replay comments do not exist
```

Comment:
I'm not sure these will be returning.

Reply:
existing issue: https://github.com/ppy/osu/issues/4796

it will still be listed since it's a difference

Reply:
Should be reworded to "niconico-style scrolling comments", since there are also beatmap page comments (which are already implemented in the game).